### PR TITLE
check inputs

### PR DESCRIPTION
**Pull request by matthiasto**
Check that max 1 entrainment and resistance shp file is given in input

### CODE REVIEW

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 1]
```diff
@@ -45,10 +45,20 @@ def readAIMECinputs(avalancheDir, dirName='com1DFA'):
 
     profileLayer = glob.glob(os.path.join(avalancheDir, 'Inputs', 'LINES',
                                           '*aimec*.shp'))
+    try:
+        message = 'There should be exactly one path_aimec.shp file containing the avalanche paths in ' + avalancheDir + '/Inputs/LINES/'
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:49]:
```suggestion
        message = 'There should be exactly one path_aimec.shp file containing (multiple) avalanche paths in ' + avalancheDir + '/Inputs/LINES/'
```
Maybe? To make put more emphasis on that you can have multiple paths in one shape file?

**Review by fso42 — APPROVED** [review]
Just as small 'maybe' comment...

File: avaframe/ana3AIMEC/ana3AIMEC.py (hunk 1 above)
Comment (matthiasto) [reply avaframe/ana3AIMEC/ana3AIMEC.py:49]: for aimec there should only be one path. For AB I agree that there can be multiple paths in the shp file

Comment (fso42) [reply avaframe/ana3AIMEC/ana3AIMEC.py:49]: ah ok, then `paths -> path`

**Pull request merged**